frontend/src/components/DelayedNextButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from "antd";
+import React, {MouseEvent, ReactElement, useState} from "react";
+import styled from "styled-components";
+import { INK, INK_HOVER } from "../constants/colors";
+
+interface NextButtonProps {
+  top?: number,
+  text?: string | '',
+  onClick?:((event: MouseEvent<HTMLInputElement>) => void),
+  icon?: ReactElement,
+  className: string,
+  delay: number
+}
+
+const ButtonContainer = styled(Button)`
+  background: ${INK};
+  border: none;
+  border-radius: 12px;
+  font-size: 16px;
+  font-weight: 700;
+  height: 50px;
+  margin-bottom: 20px;
+  width: 120px;
+
+  margin-top: ${(prop: NextButtonProps) => prop.top}px;
+  span {
+    color: #fff;
+  }
+
+  :hover {
+    background: ${INK_HOVER};
+  }
+
+  @media (max-width: 600px) {
+    width: 100%;
+  }
+`;
+
+const DelayedNextButton = ({ className = "", text, top, onClick, icon, delay}: NextButtonProps)  : ReactElement => {
```

Comment:
I think it's okay not to include className? Do we have to?

Reply:
className is needed if you want to do `styled(DelayedNextButton)`. That is how the styling works. If not, then you do not need it.